saferpay_test/src/Form/SaferpayTestForm.php

```diff
@@ -26,10 +28,67 @@ public function getFormId() {
   /**
    * {@inheritdoc}
    */
-  public function buildForm(array $form, FormStateInterface $form_state, TermInterface $taxonomy_term = NULL) {
-    $form['submit'] = array(
+  public function buildForm(array $form, FormStateInterface $form_state, Request $request = NULL) {
+    foreach ($request->query->all() as $key => $value) {
+      drupal_set_message($key . $value);
+    }
+
+    $form_elements = array(
+      'success_link' => $request->query->get('SUCCESSLINK'),
+      'fail_link' => $request->query->get('FAILLINK'),
+      'back_link' => $request->query->get('BACKLINK'),
+      'notify_url' => $request->query->get('NOTIFYURL'),
+      'status' => 'success',
+    );
+
+    $DATA= array(
+      '@MSGTYPE' => '',
```

Comment:
Please look at an example and use hardcoded example values based on what you see there. our test data should be as close to a real response as possible.